### PR TITLE
CB2-3295: EVL Feed Endpoint

### PR DESCRIPTION
**Pull request by tom-bjss**
This should not be merged until develop is ready.

The changes contain the following steps:
1. query the top database for the eve feed data from the new view
2. format this data into a new type
3. save that to a csv file as described in part by the atom documentation

## Checklist
- [x] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,15 +86,13 @@
     "fork-ts-checker-webpack-plugin": "6.0.8",
     "husky": "4.3.8",
     "jest": "26.6.3",
-    "json-serverless": "1.6.14",
```
Comment (tom-bjss) [inline package.json:85]: Nerveless wasn't being used for this projects S3 capabilities so I removed them but fixed them in my stash. Needs some refactoring though so will be in a future PR

File: src/infrastructure/s3BucketService.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import AWS from 'aws-sdk';
```
Comment (Ian-Hodges) [inline src/infrastructure/s3BucketService.ts:1]: Is this used? You don't have the serverless-s3-local plugin installed.

File: src/infrastructure/s3BucketService.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  console.log(`uploading ${fileName} to S3`);
+  s3.upload(params, (err) => {
+    if (err) {
+      console.log(err);
+    }
+    callback();
+  });
+}
+
+function configureS3() {
```
[the remark this answers is not in the document]
Comment (tom-bjss) [reply src/infrastructure/s3BucketService.ts:16]: I do plan on adding the server less stuff back in, its just the module has a high venerability with no fix just yet

[the remark this answers is not in the document]
Comment (Ian-Hodges) [reply src/infrastructure/s3BucketService.ts:16]: Yes, I had the same issue. It's a dev dependency so I manually chopped Dicer out of the package-lock.json.

**Review by naathanbrown — DISMISSED** [review]
Approved, we could maybe look at increasing the logging we have on this and using something like the logging level found on the push file lambda but not necessary.

**Review by Ian-Hodges — APPROVED** [review]

**Review by naathanbrown — APPROVED** [review]

**Pull request merged**